metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -349,7 +349,7 @@ def parse_restli_response(response):
 
 
 @ingest.command()
-@click.argument("path", type=click.Path(exists=True))
+@click.argument("path", type=click.Path(exists=False))
```

Comment:
would be good to add a comment explaining why we made this change

otherwise the exists=False feels pretty unintuitive